search-sentiments-map/routes/search.js

```diff
@@ -26,22 +26,141 @@ const json = require('./../public/country-code.json');
 global.Headers = fetch.Headers;
 
 const STALE_SEARCH_RESULT_THRESHOLD_7_DAYS_MS = 7 * 24 * 60 * 60000;
+// TODO(carmenbenitez): Set this to be 12 hours when our data is
+// updated every 12 hours.
+const CURRENT_SEARCH_RESULT_THRESHOLD_24_HOURS_MS = 24 * 60 * 60000;
 
 /** 
- * Renders a JSON array of the top search results for all countries with API data
- * obtained every 12 hours for the specified topic.
+ * Renders a JSON array of the top search results for all countries with API
+ * data obtained every 12 hours for the specified topic.
  */
 router.get('/:topic', (req, res) => {
   let topic = req.params.topic;
   retrieveSearchResultFromDatastore(topic).then(customSearchTopicJsonArray => {
-    // TODO(ntarn): Remove console.log statements when finished debugging.
-    console.log('ntarn debug: In the router.get method'); 
-    console.log(customSearchTopicJsonArray);
     res.setHeader('Content-Type', 'application/json');
     res.send(customSearchTopicJsonArray);
   });
 });
 
+/** 
+ * Renders a JSON array of the top search results for requested countries with
+ * API data from within the last 12 hours for the specified topic.
```

Comment:
Instead of specifying 12 hours, use the @code constant name. That way you wont need to worry about forgetting to update comments.